### PR TITLE
refactor: set lxcfs managed by systemd

### PR DESCRIPTION
**Pull request by CodeJuan**
### Ⅰ. Describe what this PR did
To improve availability, lxcfs process should be managed by other process, such as systemd.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Describe how you did it
1. Remove the code of starting lxcfs in Pouchd. 
2. Check if the mount point of lxcfs is existed when starting Pouchd.

### Ⅳ. Describe how to verify it
1. Make sure lxcfs is running, and the mount point of lxcfs is `/var/lib/lxcfs`
2. `pouchd --enable-lxcfs`
3. `pouch run -m 512M --enableLxcfs=true  registry.hub.docker.com/library/busybox:latest head -n 1 /proc/meminfo`
4. The output should be `524288 kB`

### CODE REVIEW

**Comment by Letty5411** [comment]
`hack/make.sh` also need to be edited to install fuse.

File: hack/package/rpm/service/lxcfs.service [hunk 1]
```diff
@@ -10,6 +10,7 @@ KillMode=process
 Restart=on-failure
 ExecStopPost=-/usr/bin/fusermount -u /var/lib/lxcfs
 Delegate=yes
+ExecStartPost=pouch remount-lxcfs
```
Comment (Letty5411) [inline hack/package/rpm/service/lxcfs.service]: also need to change `./package/deb/systemd/pouch.service`

**Comment by HusterWan** [comment]
LGTM

**Comment by xigang** [comment]
The same problem.
```
# cat /var/lib/lxc/lxcfs/proc/cpuinfo
cat: /var/lib/lxc/lxcfs/proc/cpuinfo: Transport endpoint is not connected
```
@CodeJuan @Letty5411

**Comment by xigang** [comment]
https://github.com/xigang/lxcfs-admission-webhook/blob/dev/script/container_remount_lxcfs.sh

**Pull request merged**